plugins/backend/catalog-backend-module-okta/src/providers/OktaOrgEntityProvider.ts

```diff
@@ -115,6 +119,7 @@ export class OktaOrgEntityProvider extends OktaEntityProvider {
     this.hierarchyConfig = options.hierarchyConfig;
     this.customAttributesToAnnotationAllowlist =
       options.customAttributesToAnnotationAllowlist || [];
+    this.chunkSize = options.chunkSize || 500;
```

Comment:
the default seems very high here. How many companies will have over 500 different groups?

Reply:
Ive changed it to 250.